pkg/icingadb/v1/icingadb_instance.go

```diff
@@ -18,4 +18,5 @@ type IcingadbInstance struct {
 	Icinga2EventHandlersEnabled       types.Bool      `json:"icinga2_event_handlers_enabled"`
 	Icinga2FlapDetectionEnabled       types.Bool      `json:"icinga2_flap_detection_enabled"`
 	Icinga2PerformanceDataEnabled     types.Bool      `json:"icinga2_performance_data_enabled"`
+	IcingadbVersion                   string          `json:"icingadb_version"`
```

Comment:
That json tag is kinda misleading, since there won't be any JSON value for it read from Redis. Thus, I would use a `db:...` tag instead and ignore the json tag explicitly with `json:"-"` or thereof.

Reply:
I have replaced the `json` struct tag with an empty (`json:"-"`) one. However, since the other struct fields do not contain a `db` tag and iirc the IGL's `database.columnMap` does not honor this field, I decided against adding it as well.